include/wx/desktopenv.h

```diff
@@ -0,0 +1,33 @@
+//////////////////////////////////////////////////////////////////////////////
+// Name:        wx/desktopenv.h
+// Purpose:     wxDesktopEnv class interface
+// Author:      Igor Korot
+// Created:     2021-09-05
+// Copyright:   (c) 2021 wxWidgets development team
+// Licence:     wxWidgets licence
```

Comment:
```suggestion
// Licence:     wxWindows licence
```

Applies everywhere.

Reply:
@PBfordev ,
I fixed those, however, there are 2 more files with mentioning of "wxWidgets license" - mimetype.h/cpp.

I don't know what the plan is for them to be.

Vadim?